### PR TITLE
EKF2: fix yaw variance initialization

### PR DESCRIPTION
**Pull request by bresch**
### Solved Problem
By checking the yaw variance (error around the Down axis in local frame) before and after a yaw reset I realized that something was incorrect. For example I was getting this (from the pitch90 initialization unit test):
`increase yaw var by 0.090000: 0.000335 -> 0.010335`

### Solution
Use SymForce to compute the equivalent quaternion variance from some yaw variance defined in NED
Now:
`increase yaw var by 0.090000: 0.000340 -> 0.090340`

### Changelog Entry
For release notes:
```
Fix yaw uncertainty initialization
New parameter: -
Documentation: -
```

### Test coverage
- unit test

### CODE REVIEW

File: src/modules/ekf2/EKF/ekf_helper.cpp [hunk 1]
```diff
@@ -1008,46 +1009,12 @@ void Ekf::updateGroundEffect()
...
-	P(2,3) -= yaw_variance*SQ[0]*SQ[3];
-	P(3,3) += yaw_variance*sq(SQ[3]);
-	P(1,0) += yaw_variance*SQ[1]*SQ[2];
-	P(2,0) += yaw_variance*SQ[0]*SQ[2];
-	P(2,1) += yaw_variance*SQ[0]*SQ[1];
-	P(3,0) -= yaw_variance*SQ[2]*SQ[3];
-	P(3,1) -= yaw_variance*SQ[1]*SQ[3];
-	P(3,2) -= yaw_variance*SQ[0]*SQ[3];
+	matrix::SquareMatrix<float, 4> q_variance;
+	sym::YawVarToQuatVar(getStateAtFusionHorizonAsVector(), yaw_variance, &q_variance);
```
Comment (dagar) [inline src/modules/ekf2/EKF/ekf_helper.cpp:1028]: Nitpick, maybe we should know from the signature we're only getting the lower triangle populated.

Comment (bresch) [reply src/modules/ekf2/EKF/ekf_helper.cpp:1028]: Good point

Comment (bresch) [reply src/modules/ekf2/EKF/ekf_helper.cpp]: done

**Comment by bresch** [comment]
I'll also update the tilt covariance initialization auto-code soon

**Review by dagar — APPROVED** [review]

**Pull request merged**